crates/engarde/src/syntax.rs

```diff
@@ -46,6 +50,16 @@ impl Syntax {
         themes.into_iter()
     }
 
+    fn css_class_style() -> ClassStyle {
+        ClassStyle::SpacedPrefixed { prefix: "c-" }
+    }
+
+    /** Get the content of a css file to apply the specified color theme when using the 'css' theme */
```

Comment:
This should be `///` style comment

(sorry, overlooked this commit previously)

Reply:
Too much switching between languages, this is a Typescript doc comment and not a rust one XD